src/basal_mass_balance/BMB_main.f90

```diff
@@ -502,6 +511,8 @@ SUBROUTINE remap_BMB_model( mesh_old, mesh_new, ice, BMB, region_name)
         CALL crash('Remapping after mesh update not implemented yet for parameterised BMB')
       CASE ('inverted')
         ! No need to do anything
+      CASE ('laddie')
+        ! No need to do anything
```

Comment:
Do we actually not need to do anything, or is it just not implemented yet? I would expect we'd need to re-read the Laddie BMB field from the NetCDF file

Reply:
Yes  good point, I'll add a remap routine to reread the BMB file